tiledb/query_condition.py

```diff
@@ -49,15 +49,10 @@ class QueryCondition:
 
         ``bool_term ::= bool_expr | bool_term and_op bool_expr``
```

Comment:
Let's just get rid all these metasyntax lines. We would need to completely rewrite the BNF for it to correctly match what Python AST has. It's something that will take a lot more time, but we have higher priorities to take care of than rewriting a BNF at the moment.